### DISCUSSION TITLE
wire protocol

### DISCUSSION THREAD

**Discussion by jahess** [CommandPost/CommandPost#3171] [General]
Would like to repurpose a touch bar.  Anyone ever seen or have a description of the data interface on the touch bar hardware itself?

I think I see an SPI interface in the schematic at:

   https://www.it-tech.com.au/how-to-fix-2018-macbook-pro-15-touch-bar-not-working-issue/

But knowing that, if accurate, would only be the beginning.

Thanks in advance!

**Comment by latenitefilms** [comment]
I think it would be very hard or even impossible to do. Discussed here:

https://twitter.com/sakieqatar/status/1576486769024827392?s=46&t=GIlH4Q0fa3qr7Xb2RhkfIw
